hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/PreHandleContextImpl.java

```diff
@@ -519,6 +520,21 @@ public TransactionKeys allKeysForTransaction(@NonNull TransactionBody body, @Non
         return context;
     }
 
+    @Override
+    public Key getKeyFromAccount(@NonNull final AccountID accountID) throws PreCheckException {
```

Comment:
Not a problem, but we could name it something slightly shorter

```suggestion
    public Key getAccountKey(@NonNull final AccountID accountID) throws PreCheckException {
```